GitUI/Theming/Renderers/ScrollBarRenderer.cs

```diff
@@ -50,41 +50,25 @@ private static Brush GetBackBrush(int stateId, Parts partId)
                 case Parts.SBP_THUMBBTNVERT:
                 case Parts.SBP_GRIPPERHORZ:
                 case Parts.SBP_GRIPPERVERT:
-                    switch ((States.TrackThumb)stateId)
+                    return (States.TrackThumb)stateId switch
                     {
-                        case States.TrackThumb.SCRBS_NORMAL:
-                            return SystemBrushes.ScrollBar;
-
-                        case States.TrackThumb.SCRBS_HOVER:
-                            return SystemBrushes.ControlLight;
-
-                        case States.TrackThumb.SCRBS_HOT:
-                            return SystemBrushes.ControlDark;
-
-                        case States.TrackThumb.SCRBS_PRESSED:
-                            return SystemBrushes.ControlDarkDark;
-
-                        // case States.TrackThumb.SCRBS_DISABLED:
-                        default:
-                            return SystemBrushes.Control;
-                    }
+                        States.TrackThumb.SCRBS_NORMAL => SystemBrushes.ScrollBar,
+                        States.TrackThumb.SCRBS_HOVER => SystemBrushes.ControlLight,
+                        States.TrackThumb.SCRBS_HOT => SystemBrushes.ControlDark,
+                        States.TrackThumb.SCRBS_PRESSED => SystemBrushes.ControlDarkDark,
+                        _ => SystemBrushes.Control
+                    };
 
                 // case Parts.SBP_LOWERTRACKHORZ:
                 // case Parts.SBP_LOWERTRACKVERT:
                 // case Parts.SBP_UPPERTRACKHORZ:
                 // case Parts.SBP_UPPERTRACKVERT:
                 // case Parts.SBP_SIZEBOX:
                 default:
-                    switch ((States.TrackThumb)stateId)
+                    return (States.TrackThumb)stateId switch
                     {
-                        // case States.TrackThumb.SCRBS_NORMAL:
-                        // case States.TrackThumb.SCRBS_HOVER:
-                        // case States.TrackThumb.SCRBS_HOT:
-                        // case States.TrackThumb.SCRBS_PRESSED:
-                        // case States.TrackThumb.SCRBS_DISABLED:
```

Comment:
keep?

Reply:
yes, please

Reply:
@RussKie as mentioned above I excluded these intentionally. There were quite a few changes here, mostly made via automatic codefix. I personally do not think we need comments for missing values when we have a catch-all clause. What purpose does it serve?

I don't have the apetite to go through and reinstate all these comments manually, especially given the StyleCop formatting constraints around comments. If you prefer comments, let's just close this out to keep them.

Reply:
This particular switch expression is a n-op and should just be inlined anyway.